CHANGELOG.md

```diff
@@ -6,6 +6,43 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [10.19.0]
+### Added
+- Add ENS wildcard and secure offchain resolution (ENSIP-10 & EIP3668) ([#14675](https://github.com/MetaMask/metamask-extension/pull/14675))
+- Add "What's New" notification for Add Popular Networks ([#15121](https://github.com/MetaMask/metamask-extension/pull/15121))
+- Add new translations for DE, EL, ES, FR, HI, ID, JA, KO, PT, RU, TL TR, VI and ZH ([#14975](https://github.com/MetaMask/metamask-extension/pull/14975))
+- Add Goerli, BSC, Optimism, Polygon, Avalance, Fantom and Arbitrum networks to query balances using BalanceChecker ([#13846](https://github.com/MetaMask/metamask-extension/pull/13846))
+- Swaps: on the STX status page ([#14995](https://github.com/MetaMask/metamask-extension/pull/14995))
+  - Add a link to the Swap tx
+  - Add a link for "Create a new swap"
+- Swaps: add a tx details view for cancelled Swaps on the Activity tab ([#15273](https://github.com/MetaMask/metamask-extension/pull/15273))
+- [FLASK] Add snap update metrics ([#15206](https://github.com/MetaMask/metamask-extension/pull/15206))
+
+### Changed
+- Update Confirmation screen styling by adding a top margin and moving Site Icon next to the site URL ([#15136](https://github.com/MetaMask/metamask-extension/pull/15136))
+- Remove 'ADD_POPULAR_NETWORKS' feature flag as the feature is enabled by default ([#15229](https://github.com/MetaMask/metamask-extension/pull/15229))
+- Remove dark mode and transaction insights from the "What's New" announcements ([#15279](https://github.com/MetaMask/metamask-extension/pull/15279))
+- Update design tokens package and consolidate shadow values with new tokens ([#15264](https://github.com/MetaMask/metamask-extension/pull/15264))
+- Update Zendesk ticket request URLs for a facilitating the bug reporting process ([#15458](https://github.com/MetaMask/metamask-extension/pull/15458))
+- Swaps: show a block explorer URL and hide "Add token" prompt, if block explorer URL is not available on Build Quote screen ([#15198](https://github.com/MetaMask/metamask-extension/pull/15198))
+- Swaps: Standardize the spelling of 'cancelled' ([#15266](https://github.com/MetaMask/metamask-extension/pull/15266))
+- Swaps: Performance improvements for Swaps, especially on the Build Quote page (~7.2x faster) ([#15359](https://github.com/MetaMask/metamask-extension/pull/15359))
+- [FLASK] Disable "Mark all as read button" when there are no notifications ([#15333](https://github.com/MetaMask/metamask-extension/pull/15333))
+- [FLASK] Update Snaps packages to version `^0.18.1` ([#15230](https://github.com/MetaMask/metamask-extension/pull/15230))
+  - Updates the following packages from `0.16.0` to `0.18.1`:
+    - `@metamask/execution-environments`
+    - `@metamask/rpc-methods`
+    - `@metamask/snap-controllers`
+    - `@metamask/snap-types`
+  - Serialization issues were fixed in `@metmask/execution-environments`. See the ([releases](https://github.com/MetaMask/snaps-skunkworks/releases)) of the Snaps monorepo for details.
```

Comment:
just want to confirm `package.json` contains some of these packages on version `0.20.0` 
https://github.com/MetaMask/metamask-extension/blob/bc0d32de56995945ebae873b038cbc733ff0ad96/package.json#L136

`@metamask/rpc-methods": "^0.20.0`
`@metamask/snap-controllers": "^0.20.0`
`@metamask/snap-utils": "^0.20.0`

this release will contain them on version `0.20.0` or will be `0.18.1` ?

Reply:
@BeroBurny This release will contain `0.20.0`. I have yet to update the changelog 😄

Reply:
thanks for the fast response, now need to prepare PRs for upcoming changes